### PR TITLE
Adds AA battery

### PR DESCRIPTION
**Pull request by Thunder12345**
1000 charge, non-rechargable, rather shitty battery. Can be built using a metal sheet, and 5 units each of lithium and cupric oxide. Cupric oxide can be produced by heating copper and oxygen to 700K.

:cl:
rscadd: NanoTrasen has recently released instructions on the production of AA batteries using metal and the easily obtainable chemicals lithium and cupric oxide. The latter of these can be produced by heating a mixture of copper and oxygen to extreme temperatures.
/:cl:

### CODE REVIEW

**Comment by Jazaen** [comment]
It would be great if they exploded (-1, -1, 1) after being heated, just for flavor reasons.

File: code/modules/power/cell.dm [hunk 1]
```diff
@@ -158,6 +158,24 @@
...
+	icon_state = "aacell"
+	origin_tech = null
+	maxcharge = 1000
+	chargerate = 0
+	materials = list(MAT_GLASS=40)
+	rating = 2
+
+/obj/item/weapon/stock_parts/cell/oneuse/updateicon()
+	..()
+	overlays.Cut() //Hacky meme way to remove the charge overlays until I find a better way
```
Comment (Cheridan) [inline code/modules/power/cell.dm]: :V

Comment (Thunder12345) [inline code/modules/power/cell.dm]: Find me a better way to do it and I'll do it a better way

**Comment by Fox-McCloud** [comment]
What's the point? What could I possibly use this in that's even vaguely useful?

**Comment by WJohn** [comment]
I'm with fox on this one. This is completely useless when you can more easily, more cheaply produce better batteries from r&d.

I am also profoundly against adding a new chemical with just one already outlandish use.

**Comment by octareenroon91** [comment]
Basic cells cost less material than this.

**Comment by TheNightingale** [comment]
Just 5u each of lithium, copper and oxygen would work. It lets Chemistry (+Medical) make their own power cells, if they ever need one.

**Comment by Bawhoppen** [comment]
I'm all for more variety of things even if they serve a similar purpose as something else but this one is just kind of snowflakey

**Comment by duncathan** [comment]
Make it so that you can ghetto chem it

**Comment by Iamgoofball** [comment]
# LITHIUM ION BATTERIES

please make it so I can put these in sec's equipment and make it corrode the SHIT OUT OF IT

tyvm

**Comment by Iamgoofball** [comment]
@duncathan You should be able to ghettochem it already with botany plants. I'll check it out.

**Comment by octareenroon91** [comment]
what, potatoes?

**Comment by Iamgoofball** [comment]
Potsto batteries dont hold enough charge for anything to work

They're gimmick joke items only useful for griefing borgs

**Comment by octareenroon91** [comment]
1k no-recharge batteries are pretty low down there too.

Poisoning secways sounds like good stuff though.

**Comment by Thunder12345** [comment]
Is there any way for table crafting to output more than one of the resulting item?

**Comment by coiax** [comment]
Also, there's already [rechargable AA batteries](https://github.com/tgstation/-tg-station/blob/35bfb9abb6d3e7952d14a1781c9ffc3a68d2df00/code/modules/power/cell.dm#L149) in the code.

**Comment by Razharas** [comment]
I dont think these batteries are needed but the fact that it is tablecrafting recipe made with just a few lines makes me fuzzy
Truly tablecrafting is a bliss

**Comment by KorPhaeron** [comment]
>What's the point? What could I possibly use this in that's even vaguely useful?

**Comment by coiax** [comment]
It's almost like R&D being able to shit out huge batteries at no cost makes every other source of making cells pointless. But that's just me.

**Comment by Thunder12345** [comment]
Also remember #16715 is going to leave these as the only way to produce charged batteries, if it gets merged.

**Comment by coiax** [comment]
> Also remember #16715 is going to leave these as the only way to produce charged batteries, if it gets merged.

Sources of charged batteries also include ordering them from Cargo, Botany potato batteries (and I think there's rumbling of a high power EMPotato that's gonna be added) and Xenobiology yellow slime cells.

**Comment by WJohn** [comment]
Or, you know, nutting up and putting 30 seconds towards charging one energy cell for when you plan to use it in your cattle prod or cyborg. You can do other shit while thats charging.

**Comment by coiax** [comment]
If we adjust the recipe to iron and 5u lithium, making it accessible with just basic chemistry, then it's useful in emergencies. Although, you'd get some weird arbitrage effects, given how chem dispensers probably would use less than 1000 charge making 5u of lithium. But that's _balance_

I'll :+1: this, it's cute, and might save the day once.

**Comment by KorPhaeron** [comment]
>2 weeks without activity

**Pull request closed without merging**